examples/aviator/main.rs

```diff
@@ -19,7 +19,7 @@ const COLOR_BROWN_DARK: three::Color = 0x23190f;
 const COLOR_BLUE: three::Color = 0x68c3c0;
 
 fn main() {
-    env_logger::init().unwrap();
+    env_logger::init();
```

Comment:
What about enabling logging for every example?

Reply:
would be nice, but requires some sort of example boilerplate, and creeps out of scope for this PR